src/WebJobs.Extensions.DurableTask/OrchestrationRuntimeStatus.cs

```diff
@@ -11,6 +11,11 @@ namespace Microsoft.Azure.WebJobs
     /// </summary>
     public enum OrchestrationRuntimeStatus
     {
+        /// <summary>
+        /// The status of the orchestration could not be determined.
+        /// </summary>
+        Unknown = -1,
```

Comment:
I take it this is unrelated to this fix, but more related to [this PR ](https://github.com/Azure/durabletask/pull/277#pullrequestreview-223391133)?

Reply:
Yes, this is related to the other PR.